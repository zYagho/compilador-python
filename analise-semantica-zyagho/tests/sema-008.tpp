{Erro: Variável 'a' não declarada}
{Aviso: Variável 'b' declarada e inicializada, mas não utilizada}
{Erro: Função 'principal' deveria retornar 'inteiro', mas retorna 'vazio'}

inteiro principal()
	flutuante: b

	b := a + 1.0

fim
