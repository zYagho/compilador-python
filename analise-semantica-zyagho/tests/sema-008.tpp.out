ERR-SEM-VAR-NOT-DECL
WAR-SEM-VAR-DECL-INIT-NOT-USED
ERR-SEM-FUNC-RET-TYPE-ERROR
